src/main.cpp

```diff
@@ -760,7 +760,7 @@ bool CheckFinalTx(const CTransaction &tx, int flags)
  * 2. P2SH scripts with a crazy number of expensive
  *    CHECKSIG/CHECKMULTISIG operations
  */
-bool AreInputsStandard(const CTransaction& tx, const CCoinsViewCache& mapInputs)
+bool AreInputsStandard(const CTransaction& tx, const CCoinsViewCache& mapInputs, uint32_t consensusBranchId)
```

Comment:
was sure I saw this change in one of the other NU0 Prs, maybe transaction format

Reply:
This PR includes the transaction format commits, so any changes from that will disappear once that PR is merged and I can rebase this on master.

Reply:
Yeah, but this appears in one of the sighash commits  https://github.com/str4d/zcash/commit/bb1275326639cbf35a76ff9c9f4e1daa0bd64b3e

Reply:
No big deal, wondering if it should have been in transaction format